test/e2e/density.go

```diff
@@ -162,21 +162,18 @@ var _ = Describe("Density", func() {
 		// (metrics from other tests affects this one).
 		// TODO: Reenable once we can measure latency only from a single test.
```

Comment:
it seems like we should either remove this TODO or leave the option to make these tests unskipped by default? @fgrzadkowski